dist/style.css

```diff
@@ -0,0 +1,167 @@
+body {
+    background-color: #a2d5f2;
+    font-family: 'Play', sans-serif;
+    text-align: center;
+}
+
+input:focus::placeholder {
+    color: transparent;
+}
+
+#players {
+    display: grid;
+    grid-gap: 10px;
+    grid-template-columns: repeat(3, 1fr);
+    grid-template-rows: repeat(6, 1fr);
+    grid-template-areas:
+        "j k l"
+        "m n o"
+        "p q r"
+        "a b c"
+        "d e f"
+        "g h i";
```

Comment:
you could use grid-template-rows and grid-template-columns instead of defining 18(!) different grid-areas